src/Controller/PageAdminController.php

```diff
@@ -74,38 +75,48 @@ public function __construct(
      * @throws AccessDeniedException
      * @throws NotFoundHttpException
      */
-    public function composeContainerShowAction(Request $request = null): Response
+    public function composeContainerShowAction(?Request $request = null): Response
```

Comment:
```suggestion
    public function composeContainerShowAction(Request $request): Response
```
so no need to check it after